Texstyle/Texstyle/TextStyle.swift

```diff
@@ -57,6 +66,13 @@ open class TextStyle {
         }
     }
 
+    /// Use this attribute to specify the shadow.
+    public var shadow: NSShadow? {
```

Comment:
If I'm not mistaken there are a shadow wrapper in text attributes. What the purpose of this property?

Reply:
What's the shadow wrapper? I just added missing attributes for all keys.

Reply:
Looks like I missed `ShadowStyle` implementation. I've added [an issue(https://github.com/rosberry/texstyle/issues/12)] for it. Let's add this `NSShadow` implementation for now if you don't mind.